incubator/fluentd/templates/configmap.yaml

```diff
@@ -9,16 +9,15 @@ metadata:
     helm.sh/chart: {{ include "fluentd.chart" . }}
 apiVersion: v1
 data:
-  {{- if .Values.elasticsearch.enabled }}
-  custom.conf: |
-    {{ .Values.elasticsearch.customConf | nindent 4 }}
-  {{- else }}
   fluent.conf: |
     {{ tpl (.Files.Get "files/kubernetes.conf") . | nindent 4 }}
+    {{ .Values.additional_filters_and_sources | nindent 4 }}
```

Comment:
Will it cause any issues if this outputs a blank line in the config when the value is empty?

Reply:
Good question, but nope, its xml so indentation doesn't even really matter.